sql-delta-import/src/main/scala/com/razorpay/spark/jdbc/JDBCImport.scala

```diff
@@ -127,7 +128,16 @@ class JDBCImport(
 
     val database = importConfig.database
 
-    val connectionUrl = s"jdbc:$dbType://$host:$port/$database"
+    val urlAdditionalArgs = if (dbType == Constants.MYSQL) {
+      if (importConfig.maxExecTimeout.isDefined) {
+        s"?sessionVariables=MAX_EXECUTION_TIME=${importConfig.maxExecTimeout.get}"
+      } else {
+        ""
+      }
+    } else {
+      ""
+    }
+    val connectionUrl = s"jdbc:$dbType://$host:$port/$database" + urlAdditionalArgs
```

Comment:
why not just add it as a property here? https://github.com/razorpay/sqoop-on-spark/blob/master/sql-delta-import/src/main/scala/com/razorpay/spark/jdbc/JDBCImport.scala#L112